nixos/release.nix

```diff
@@ -315,7 +315,7 @@ rec {
         modules = [
           configuration
           versionModule
-          ./maintainers/scripts/ec2/amazon-image.nix
```

Comment:
We should probably add a comment about it's deprecation if that's your plan?

Not sure if could outright kill it because there's probably(?) people using `amazonImageZfs` and we don't really have good story for that with repart. 

Having `amazonImage` and `amazonImageZfs` while both depend on completely different implementations sound confusing for users.

Reply:
Amazon image zfs hasn't been uploaded in 4 years. I see no need to keep it around. Only users are people building and uploading them themselves.  I can add a release note that zfs will be removed.